schematic/visualization/attributes_explorer.py

```diff
@@ -26,10 +28,15 @@ def __init__(
         data_model_grapher: Optional[DataModelGraph] = None,
         data_model_graph_explorer: Optional[DataModelGraphExplorer] = None,
         parsed_data_model: Optional[dict] = None,
+        graph_data_model: Optional[nx.MultiDiGraph] = None,
+        data_model_graph_pickle: Optional[str] = None,
     ) -> None:
         self.path_to_jsonld = path_to_jsonld
 
         self.jsonld = load_json(self.path_to_jsonld)
+        if data_model_graph_pickle and not graph_data_model:
+            with open(data_model_graph_pickle, "rb") as file:
+                graph_data_model = pickle.load(file)
```

Comment:
If I am reading this code correctly, doesn't this not accomplish anything?

This is being set on L53
`self.graph_data_model = data_model_grapher.graph`


On this L39 you are updating the attribute passed in if None:
`graph_data_model = pickle.load(file)`

But then `graph_data_model` is not actually being used anywhere to set anything on the `AttributesExplorer` class instance that was instantiated.

Reply:
Nice catch, thanks. I updated this to set self.graph_data_model on L37 and inside `if not data_model_grapher` on L47